pkg/host-timesync-daemon/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 		d := time.Date(int(t.tmYear+1900), time.Month(t.tmMon+1), int(t.tmMday), int(t.tmHour), int(t.tmMin), int(t.tmSec), 0, time.UTC)
 		log.Printf("Setting system clock to %s", d)
 		tv := syscall.Timeval{
-			Sec:  int64(d.Unix()),
```

Comment:
Could this one have been for non-x86 or 32-bit arches? I know we have similar casts in other repositories (but if it's deliberate; we need to add a nolint comment with an explanation

Reply:
I am not sure. I think that as long as it is not complaining, let's leave it as is. If someone has an issue somewhere on a 32-bit arch, let's worry about it then.